README.md

```diff
@@ -1,8 +1,8 @@
-# CDT GDB Debug Adapter Extension for Visual Studio Code
+# CDT GDB Debug Adapter Extension for vscode
 
-This is a Visual Studio Code extension that supports debugging using gdb and any other debugger that supports the MI protocol. It is built by the experts that provide the gdb support in the Eclipse C/C++ IDE (CDT).
+This is an extension for vscode that supports debugging using gdb and any other debugger that supports the MI protocol. It is built by the experts that provide the gdb support in the Eclipse C/C++ IDE (CDT).
```

Comment:
Should probably say `VS Code` instead.

Reply:
... if you want the trademark name there